### ISSUE TITLE
Learning rate metric inconsistent depending on optimizer

### ISSUE THREAD

**Issue by b-fg** [tensorflow/agents#883]
When using the `tf.compat.v1.train.AdamOptimizer` I can see in the TF metrics that the learning rate is the one I set. However, when using `tf.keras.optimizers.Adam` the metric always displays -1. Could you please double check this? Thanks!
